test/sharness/t0119-prometheus-data/prometheus_metrics

```diff
@@ -250,6 +250,5 @@ process_resident_memory_bytes
 process_start_time_seconds
 process_virtual_memory_bytes
 process_virtual_memory_max_bytes
-provider_reprovider_provide_count
-provider_reprovider_reprovide_count
 target_info
+total_provide_count_total
```

Comment:
Would this be better named, `total_provide_count` or `provide_count_total`?

Reply:
Sadly the OpenTelemetry exporter adds `_total` suffix automatically (prometheus adapter) and metric in `kad-dht` has `total_` prefix, so we end up with this name here.

We could fix it in `kad-dht` upstream, but we need to document it in release notes:
- it was already released in go-libp2p-kad-dht v0.35.0 (Oct 1, 2025)
- the `total_provide_count_total` metric was documented in kubo release notes of v0.38  where we announced opt-in `Provide.DHT.SweepEnabled`


Good news is that name is not consistent with the rest of kad-dht – to cleanup, I've opened PR:  
- https://github.com/libp2p/go-libp2p-kad-dht/pull/1195 – mind taking a look @gammazero? If so, we can merge that PR and then update Kubo docs and changelog.